Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/EmulationFragment.java

```diff
@@ -168,12 +172,18 @@ public void surfaceChanged(SurfaceHolder holder, int format, int width, int heig
 	@Override
 	public void surfaceDestroyed(SurfaceHolder holder)
 	{
-		Log.debug("[EmulationFragment] Surface destroyed.");
-		NativeLibrary.SurfaceDestroyed();
-
-		if (mEmulationState.isRunning())
+		if (mSurface == null)
 		{
-			pauseEmulation();
+			Log.debug("[EmulationFragment] Received surfaceDestroyed callback, ignoring.");
+		}
+		else {
```

Comment:
Doesn't our code style JAR put the brace on the next line?

Reply:
Oops yeah, I didn't have the style on this machine. Fixed.